src/layout/Footer.tsx

```diff
@@ -15,39 +18,263 @@ export const Footer = () => {
+              color: 'primary.main',
+            }}>
+            {t('footer.about_sikt')}
+          </Typography>
+
+          <MuiLink
+            sx={{ color: 'primary.main', display: 'flex', gap: '0.5rem' }}
+            data-testid={dataTestId.footer.aboutSikt}
+            target="_blank"
+            rel="noopener noreferrer"
+            href={'https://sikt.no/'}>
+            {t('footer.read_more')}
+            <LaunchIcon fontSize="small" />
+          </MuiLink>
+        </Box>
+
+        <Divider sx={{ bgcolor: 'primary.main', mb: '0.5rem' }} />
+
+        <Typography
+          sx={{
+            color: 'primary.main',
+            gridColumn: { xs: 1, lg: 2 },
+            justifySelf: 'start',
+          }}>
+          {t('about.footer_text')}
+        </Typography>
+      </div>
+
+      <div>
+        <Box sx={{ display: 'flex', gap: '0.5rem', mb: '0.5rem', fontWeight: 'bold' }}>
+          <MuiLink
+            sx={{
+              color: 'primary.main',
+              display: 'flex',
+              gap: '0.5rem',
+            }}
+            data-testid={dataTestId.footer.helpPage}
+            target="_blank"
+            rel="noopener noreferrer"
+            href={'https://sikt.no/tjenester/nasjonalt-vitenarkiv-nva/hjelpeside-nva'}>
+            {t('footer.help_page')}
+            <LaunchIcon fontSize="small" />
+          </MuiLink>
+        </Box>
+
+        <Box sx={{ display: 'flex', gap: '0.5rem', mb: '1rem', fontWeight: 'bold' }}>
+          <MuiLink
+            sx={{
+              color: 'primary.main',
+              display: 'flex',
+              gap: '0.5rem',
+            }}
+            data-testid={dataTestId.footer.systemStatusLink}
+            target="_blank"
+            rel="noopener noreferrer"
+            href={'https://sikt.no/driftsmeldinger'}>
+            {t('footer.system_status_link')}
+            <LaunchIcon fontSize="small" />
+          </MuiLink>
+        </Box>
+
+        <Divider sx={{ bgcolor: 'primary.main', mb: '0.8rem' }} />
+
+        <Box
+          sx={{
+            display: 'flex',
+            justifySelf: { xs: 'center', lg: 'start' },
+            gap: '0.5rem',
+            gridColumn: { xs: 1, lg: 3 },
+            mb: '0.3rem',
+          }}>
+          <ArrowRightAltIcon />
+          <MuiLink
+            sx={{
+              color: 'primary.main',
+              display: 'flex',
+              gap: '0.5rem',
+            }}
+            data-testid={dataTestId.footer.newsLink}
+            target="_blank"
+            rel="noopener noreferrer"
+            href={'https://pub.dialogapi.no/s/MjI2NDQ6MWVmZmQxOTktMGVkZi00YjZhLWE2ZjQtNzNmZWU2NjI2ZmFi'}>
+            {t('footer.news_link')}
+            <LaunchIcon fontSize="small" />
+          </MuiLink>
+        </Box>
+
+        <Box
+          sx={{
+            display: 'flex',
+            justifySelf: { xs: 'center', lg: 'start' },
+            gap: '0.5rem',
+            gridColumn: { xs: 1, lg: 3 },
```

Comment:
ser ikke ut som at disse har noen effekt?
```suggestion
            gap: '0.5rem',
```